test/ranges.jl

```diff
@@ -887,7 +887,7 @@ function range_fuzztests(::Type{T}, niter, nrange) where {T}
         @test m == length(r)
         @test strt == first(r)
         @test Δ == step(r)
-        @test_skip stop ≈ last(r)
+        @test stop ≈ last(r) atol = eps((n-1)*Δ) + eps(stop)
```

Comment:
@simonbyrne your explanation in the PR seems like a good comment
```suggestion
        @test stop ≈ last(r) atol = eps((n-1)*Δ) + eps(stop) # the tolerance needs to take into account the intermediate rounding when computing stop
```